Rules

```diff
@@ -43,6 +43,10 @@ compile "/code/**/*" do
   filter :ably_jsbin_publisher
 end
 
+compile "/partials/**/_*" do
```

Comment:
@tomczoink please see https://wiki.ably.io/pages/Git-best-practice.html in regards to git commit messages.  Anyone (including me) looking at this commit won't understand why this change was needed.  You are only stating in the commit message what we can tell from the code itself.  Why was this changed done, why is it needed?